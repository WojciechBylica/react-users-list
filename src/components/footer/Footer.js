import './style.css';

const Footer = () => (
  <footer className="footer">
    <p>Written for learning</p>
  </footer>
);

export default Footer;
