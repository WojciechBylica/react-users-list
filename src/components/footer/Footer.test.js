import React from 'react';
import renderer from 'react-test-renderer';
import Footer from '.';

it('should render footer', () => {
  const tree = renderer.create(<Footer />);
  expect(tree).toMatchSnapshot();
});
