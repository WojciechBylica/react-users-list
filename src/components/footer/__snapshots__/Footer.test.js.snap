// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render footer 1`] = `
<footer
  className="footer"
>
  <p>
    Written for learning
  </p>
</footer>
`;
